SizeMatters/SizeMatters/ModConfig.cs

```diff
@@ -22,6 +22,8 @@ public class ModConfig
 
         public float TonnageCapMin = 20f;
         public float TonnageCapMax = 150f;
+        
+        public Dictionary<string, float> UnitTonnageRatio = new Dictionary<string, float>();
```

Comment:
If you want default tags when mod.json don't have the UnitTonnageRatio key, make sure to initialize the tags in the Config.Init() method.

Reply:
The intention was to only use it a tag if specifically configured in the config file, as to not accidentally change logic for any other mod users. So I left it without default values on purpose.